src/main/resources/label.yml

```diff
@@ -70,11 +71,13 @@ labelRules:
     properties:
       - "{$NUMERATION}{, $NAME}{, $TITLES}{, $NAME_ALTERNATIVE}{, $DATE}{ -- $FORM_SUBDIVISION}{ -- $GENERAL_SUBDIVISION}{ -- $CHRONOLOGICAL_SUBDIVISION}{ -- $GEOGRAPHIC_SUBDIVISION}"
   - types: CONCEPT, JURISDICTION
+    addLabelProperty: true
     properties:
-      - NAME
+      - "{$NAME}{, $SUBORDINATE_UNIT}{, $PLACE}{, $DATE}{ - $FORM_SUBDIVISION}{ - $GENERAL_SUBDIVISION}{ - $CHRONOLOGICAL_SUBDIVISION}{ - $GEOGRAPHIC_SUBDIVISION}"
   - types: CONCEPT, ORGANIZATION
+    addLabelProperty: true
     properties:
-      - NAME
+      - "{$NAME}{, $SUBORDINATE_UNIT}{, $PLACE}{, $DATE}{ - $FORM_SUBDIVISION}{ - $GENERAL_SUBDIVISION}{ - $CHRONOLOGICAL_SUBDIVISION}{ - $GEOGRAPHIC_SUBDIVISION}"
```

Comment:
Here also. Please use two dashes ("--") instead of one dash ("-") as in line 72.

Reply:
Done.